### PR TITLE
Bug/027 app config.js pointing at wrong directory

### PR DESCRIPTION
**Pull request by mcgonagle**

### CODE REVIEW

File: recipes/ux.rb [hunk 1]
```diff
@@ -36,62 +36,62 @@
...
+  cwd node['rbenv']['user_home']
+  user node['jellyfish']['user']
   code <<-EOH
   unzip ux-master.zip
   EOH
-  creates '/home/jellyfish/ux-master'
+  creates "#{node['rbenv']['user_home']}/ux-master"
 end
 
 bash 'mv ux-master ux' do
```
Comment (chriskacerguis) [inline recipes/ux.rb]: We should be able to remove this whole section by updating the unzip block above to use this code (will need some tweaking)

```
unzip ux-master.zip -d "/home/jellyfish/ux"
```

**Comment by chriskacerguis** [comment]
@mcgonagle Looks like this can't be auto-merged, can you please take a look?

File: recipes/ux.rb [hunk 2]
```diff
@@ -36,62 +36,53 @@
...
 
 log 'Unzip ux-master and move it to ux'
 bash 'unzip ux-master.zip' do
-  cwd '/home/jellyfish'
-  user 'jellyfish'
+  cwd node['rbenv']['user_home']
+  user node['jellyfish']['user']
   code <<-EOH
-  unzip ux-master.zip
+  /usr/bin/unzip ux-master.zip; mv ux-master ux
```
Comment (chriskacerguis) [inline recipes/ux.rb]: Why add the full path on unzip and not on mv?

File: recipes/ux.rb [hunk 3]
```diff
@@ -36,62 +36,53 @@
...
-  cwd '/home/jellyfish/ux'
+  cwd "#{node['rbenv']['user_home']}/ux"
   code <<-EOH
   /usr/bin/npm link
   /usr/bin/npm install -g
   /usr/bin/gulp production
   /usr/bin/npm install forever  -g
   EOH
-  creates '/home/jellyfish/ux/node_modules/winston'
+  creates "#{node['rbenv']['user_home']}/ux/node_modules/winston"
```
Comment (chriskacerguis) [inline recipes/ux.rb]: Why are you creating that dir (what's the purpose)?

File: recipes/ux.rb (hunk 2 above)
Comment (mcgonagle) [inline recipes/ux.rb]: Thought mv was a shell built-in. Just looked and they are not. Will add full path /bin/mv

**Comment by chriskacerguis** [comment]
:+1:

**Pull request merged**